.mergify.yml

```
@@ -16,6 +16,9 @@ pull_request_rules:
       - status-success~=^test \([0-9]+\.x, ubuntu-latest\)$
       - base=master
     actions:
+      review:
```

I think YML format will need the dashes for lists of objects:``
```suggestion
      - review:
```